### PR TITLE
Signup form localization

### PR DESCRIPTION
**Pull request by alanmoo**

### CODE REVIEW

File: components/newsletter-signup/SignupForm.jsx [hunk 1]
```diff
@@ -60,31 +64,32 @@ var SignupForm = React.createClass({
     if (this.state.validationErrors.length) {
       return (
         <div className="alert alert-danger" role="alert">
-          <p className="error-msg">Please enter an email address.</p>
+          <p className="error-msg"><FormattedMessage id="email-error" default="Please enter an email address." /></p>
```
Comment (alicoding) [inline components/newsletter-signup/SignupForm.jsx]: `defaultMessage` instead of `default`

File: components/newsletter-signup/SignupForm.jsx [hunk 2]
```diff
@@ -60,31 +64,32 @@ var SignupForm = React.createClass({
...
   },
   render: function() {
     var idPrefix = this.props.idPrefix;
+    var privacy_policy_link = <a href="https://www.mozilla.org/privacy/websites/"><FormattedMessage id="privacy_policy" default="privacy policy"/></a>
     return (
       <form className="mailinglist-signup" action={process.env.NEWSLETTER_MAILINGLIST_URL} method="POST" onSubmit={this.handleSubmit}>
         <fieldset>
           <label htmlFor={idPrefix+"privacy"} className="sr-only">
-            I'm okay with you handling this info as you explain in your <a href="https://www.mozilla.org/en-US/privacy/websites/">privacy policy</a>
+            <FormattedMessage id="privacy_consent" values={{privacy_policy_link: privacy_policy_link}} default="I'm okay with you handling this info as you explain in your {privacy_policy_link}" />
```
Comment (alicoding) [inline components/newsletter-signup/SignupForm.jsx]: same here.

File: components/newsletter-signup/SignupForm.jsx [hunk 3]
```diff
@@ -60,31 +64,32 @@ var SignupForm = React.createClass({
...
           </label>
           <input id={idPrefix+"privacy"} type="checkbox" className="sr-only" checked readOnly required />
           <p className="pp-note">
-            &#10003; I'm okay with you handling this info as you explain in your <a href="https://www.mozilla.org/en-US/privacy/websites/">privacy policy</a>.
+            &#10003; <FormattedMessage id="privacy_consent" values={{privacy_policy_link: privacy_policy_link}} default="I'm okay with you handling this info as you explain in your {privacy_policy_link}" />
```
Comment (alicoding) [inline components/newsletter-signup/SignupForm.jsx]: and here

File: components/newsletter-signup/SignupForm.jsx [hunk 4]
```diff
@@ -60,31 +64,32 @@ var SignupForm = React.createClass({
...
           </p>
-          <label htmlFor={idPrefix+"email"} className="sr-only">email</label>
+          <label htmlFor={idPrefix+"email"} className="sr-only"><FormattedMessage id="email" default="email"/></label>
           <div className="icon-field-container">
```
Comment (alicoding) [inline components/newsletter-signup/SignupForm.jsx]: here

**Comment by alicoding** [comment]
r+ with those fixes

**Comment by alanmoo** [comment]
Fixed. Waiting on Travis

**Pull request merged**